api/capi/include/nnstreamer-capi-private.h

```diff
@@ -351,6 +351,15 @@ char* ml_nnfw_to_str_prop (ml_nnfw_hw_e hw);
  */
 const char* ml_get_nnfw_subplugin_name (ml_nnfw_type_e nnfw);
 
+/**
+ * @brief Gets the element of pipeline itself (GstElement).
+ * @details With the returned reference, you can use GStreamer functions to handle the element in pipeline.
+ *          Note that caller should release the returned reference using gst_object_unref().
+ * @since_tizen 6.0
```

Comment:
This is internal private API. do not add "@since_tizen".

Reply:
updated